packages/app-runtime/test/runtime/Offboarding.test.ts

```diff
@@ -0,0 +1,59 @@
+import { AppRuntime, AppRuntimeServices } from "../../src";
+import { TestUtil } from "../lib";
+
+describe("Offboarding", function () {
+    let runtime: AppRuntime;
+
+    let services1: AppRuntimeServices;
+    let localAccount2Id: string;
+    let device2Id: string;
+
+    beforeAll(async function () {
+        // as we can't pop up multiple runtimes we have to allow multiple accounts with
+        // the same address to test offboarding
+        const configOverride = { allowMultipleAccountsWithSameAddress: true };
+        runtime = await TestUtil.createRuntime(configOverride);
+        await runtime.start();
+
+        const [localAccount1] = await TestUtil.provideAccounts(runtime, 1);
+        services1 = await runtime.getServices(localAccount1.id);
+
+        const createDeviceResult = await services1.transportServices.devices.createDevice({});
+        device2Id = createDeviceResult.value.id;
+
+        const onboardingInfoResult = await services1.transportServices.devices.getDeviceOnboardingInfo({ id: createDeviceResult.value.id });
+
+        const localAccount2 = await runtime.accountServices.onboardAccount(onboardingInfoResult.value);
+        localAccount2Id = localAccount2.id;
+        const services2 = await runtime.getServices(localAccount2.id);
+
+        await services2.transportServices.account.syncDatawallet();
+        await services1.transportServices.account.syncDatawallet();
+    });
+
+    afterAll(async function () {
+        await runtime.stop();
+    });
+
+    test("delete account", async function () {
```

Comment:
Just one tiny thing: I'd like to check that there were 2 accounts before to make it really obvious that after calling deleteAccount one account is gone. 

Then it looks good to me.

Reply:
but then I'd test that onboard account creates one account... IMO that's out of scope in the offboarding

Reply:
okely schmokely :)